resolwe/flow/managers/state.py

```diff
@@ -0,0 +1,234 @@
+"""Synchronized singleton-state container and runtime constants for the manager's needs."""
+
+# This module should not import anything local, or there will be circular dependencies,
+# since the constants are needed in various sub-modules inside resolwe.flow.managers.
+
+import json
+from collections import namedtuple
+
+import redis
+
+from django.conf import settings
+
+ManagerChannelPair = namedtuple('ManagerChannelPair', ['queue', 'queue_response'])
+
+MANAGER_CONTROL_CHANNEL = 'control'
```

Comment:
Why have these constant definitions if they are all overridden by `update_constants` below? Also why keep these things as globals, wouldn't it be better to store them in in a class instance?

Reply:
I have already off github and received no reply. In short, these are constant constants for all intents and purposes, are used as such and it would make no sense to change their nature. The fact that they are overridden is an artefact of how tests work and how prolific monkey patching is throughout the testing support code.